content/docs/user-guide/remote-objects.md

```diff
@@ -13,10 +13,19 @@ operations apply to any [object type] and location.
 [mlem objects]: /doc/user-guide/basic-concepts#mlem-objects
 [object type]: /doc/user-guide/basic-concepts#mlem-object-types
 
+## Remote MLEM projects
+
+Although you can store MLEM objects in any location such as a Git repo, Cloud
+storage, or external drives, creating a MLEM project lets you organize and
+[discover](#listing-objects) MLEM objects consistently.
+
+To create a MLEM project in a remote location, you can provide its URL or path
+to `mlem init`.
+
 ## Listing objects
 
-You can list MLEM objects inside a remote MLEM project (e.g. in a Git
-repo) with `mlem list`. There's no need to download/clone the project.
+You can list MLEM objects inside a remote MLEM project (e.g. in a Git repo) with
+`mlem list`. There's no need to download/clone the project.
```

Comment:
Question @aguschin : There is some missing link between creating a remote project and having objects in there. What if there are objects in a remote location already, can you start a project on top of them and get them indexed automatically? I'm trying to figure out the motivation behind accepting URLs in `mlem init` and how that is relevant for this guide. Thanks

Reply:
> What if there are objects in a remote location already, can you start a project on top of them and get them indexed automatically?

No, currently they won't be indexed if you start a project in that location. But we should implement this I think.

Reply:
But we're discussing removing the indexing functionality from mlem completely, so let's write it as it is. Once we do that, we'll update the docs substantially anyway.

Reply:
OK thanks for that answer. It's still unclear to me why `mlem init` accepts URLs. If you have more context please share and/or update this page again. Merging the changes so far for now.